atom-client/src/main/resources/webapp/messages/messages.properties

```diff
@@ -9,4 +9,7 @@ atomclient.task.feed.change.published_date=Published Date
 atomclient.task.feed.change.updated_date=Updated Date
 atomclient.task.feed.change.raw_content=Raw Content
 atomclient.task.feed.change.extracted_content=Extracted Content
+atomclient.task.feed.read.current_url= Current OpeMRS Feed Url
+atomclient.task.feed.read.last_url= Recent OpeMRS Feed Url
```

Comment:
if this is only for OpenMRS, name it properly and split this properties to new group for OpenMRS